### ISSUE TITLE
Cerebellar tonsil ectopia

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7728]
**Preferred term label:**
Cerebellar tonsil ectopia

**Synonyms**

**Definition (free text, please give PubMed ID)**
 an inferior location of the cerebellar tonsils below the margins of the foramen magnum.
Todo -- distinguish from  Chiari I -- Type 1 Chiari malformation (CM-I) has been historically defined by cerebellar tonsillar position (TP) greater than 3–5 mm below the foramen magnum (FM). 
See https://www.caringmedical.com/prolotherapy-news/cerebellar-tonsillar-ectopia-herniation-and-chiari-1-malformation/

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
603387 
PMID: 22729224  8/13

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Looking again at the publication, I think this is not different from Chiari malformation. closing...
